internal/app/model/entities.go

```diff
@@ -58,3 +55,11 @@ func (p *Parcel) ValidateParcelInput() error {
 
 	return nil
 }
+
+// Validates carrier request input credentials
+func (cr *CarrierRequest) ValidateCarrierId() error {
```

Comment:
should write test case for this

Reply:
The test case has already been written in handler_test.go file on line 294.